config/crd/bases/extensions.hive.openshift.io_agentclusterinstalls.yaml

```
@@ -224,6 +224,9 @@ spec:
                       TODO: Add other useful fields. apiVersion, kind, uid?'
                     type: string
                 type: object
+              mastersSchedulable:
+                description: Set to true to allow control plane nodes to be schedulable
```

Just to match the surrounding documentation, plus some extra information:
```suggestion
                description: mastersSchedulable indicates whether or not to allow control plane nodes to schedule pods. Default is false.
```

---

This line was taken from the official OpenShift page documenting this property

---

even if this line is from the bible, it needs to match conventions in this file :wink:
at the very least, give it the same structure as other description fields of this file, and keep the wording